src/unwind_dwfl.cc

```diff
@@ -150,9 +153,16 @@ static DDRes add_symbol(Dwfl_Frame *dwfl_frame, UnwindState *us) {
       return ddres_warn(DD_WHAT_UW_ERROR);
     }
   }
-  // Now we register
-  if (IsDDResNotOK(add_dwfl_frame(us, dso, pc, ddprof_mod, file_info_id))) {
-    return ddres_warn(DD_WHAT_UW_ERROR);
+
+  if (ddprof_mod->_symbol_method == kRuntimeSymbol) {
```

Comment:
@gleocadie you need to figure out how to switch between dwarf and attempting the symbol maps